### ISSUE TITLE
Adding configurable shortcuts to avoid "View/Type individual entries"

### ISSUE THREAD

**Issue by Purtl** [firecat53/keepmenu#139]
Heyho,
what do you think about the following "scenario":
* opening keepmenu
* highlighting the intended entry
* using a some kind <C-p> to only type the password/Note/name/whatever of the chosen entry. 

with an entry in the config like:
[shortcuts]
password: <C-p>
title: <C-t>
username: <C-n>
url: <C-u>
...

At the moment that is (correct me if i'm wrong) only possible by:
* opening keepmenu
* highlighting "View/Type individual entries"
* enter
* highgliht intended entry
* enter
* highlight field
* enter

**Comment by firecat53** [comment]
That already exists :smile: Look at the Auto Type sequences in the docs. If you always only want to type the password without the username or hitting enter and nothing else, edit the entry, select `Autotype` and enter `{PASSWORD}`.

Does that match what you're looking for?

**Comment by vaygr** [comment]
I guess the biggest part of the request here is shortcuts, so with different shortcuts you could have different autotype sequences.

**Comment by firecat53** [comment]
I think I see what you're saying. I'll have to think about if that's something I want to pursue.

In the meantime, you could possibly work around that using the `-a` flag to keepmenu. For example, you can have one keyboard shortcut that calls keepmenu like normal, and a second shortcut that calls `keepmenu -a '{TOTP}'` to only type the TOTP values.

**Comment by vaygr** [comment]
Yeah, exactly. TOTP is a special case though. If I have 800 entries and TOTP ones are only 50 of them, it makes little sense to output all for filtering using dmenu. That's why I came up with #132.

**Comment by firecat53** [comment]
So you'd basically have to create "temporary" duplicate entries with the different autotype values when generating the list to display to dmenu. I'm thinking that could work similar to how I have done the multiple URLs feature. That way the configuration file stays simple and only the autotype entries would need to be edited.

**Comment by vaygr** [comment]
Precisely. So keepmenu has to be smart to see if there's *only* `{TOTP}` in autotype, generate the list only of those entries that contain it.

**Comment by sf467** [comment]
> I think I see what you're saying. I'll have to think about if that's something I want to pursue.
> 
> In the meantime, you could possibly work around that using the `-a` flag to keepmenu. For example, you can have one keyboard shortcut that calls keepmenu like normal, and a second shortcut that calls `keepmenu -a '{TOTP}'` to only type the TOTP values.

Hi, I'm having a problem with keepmeun opening very slowly when I run it with the -a or -C parameter, is this normal?

**Comment by firecat53** [comment]
Definitely not normal. Make sure all keepmenu processes are killed and test `wl-copy` and `wl-paste` on the command line. With no other reports and being unable to replicate, I'm going to suspect an issue with your system. If you continue to have trouble, open a new issue please, as it's not relevant to this one. Thanks!
